docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -4,20 +4,74 @@ description: List of operations that users can perform on Codacy depending on th
 
-To review the permissions granted by each role, see the tables for each Git provider:
+To update a member's role on Codacy, update that member's role on your Git provider. When next logging in to Codacy, the member is assigned the new role.<!-- TODO is this task rather than intro/concept? --> 
 
--   [GitHub](#permissions-for-github)
--   [GitLab](#permissions-for-gitlab)
--   [Bitbucket](#permissions-for-bitbucket)
+To review the permissions granted by each role, see the tables for each Git provider:
 
-Additionally, you can grant some administrative permissions to any organization member independently of the member's role on the Git provider, using the [organization manager](#the-organization-manager-role) role.
+-   [Permissions for GitHub](#permissions-for-github)
+-   [Permissions for GitLab](#permissions-for-gitlab)
+-   [Permissions for Bitbucket](#permissions-for-bitbucket)
 
 To list and manage the members of your Codacy organization, see the [Managing people](managing-people.md) page.
 
+## The organization manager role
+
+To enable other members to manage organization settings, organization admins can share some of their permissions with any organization member using the organization manager role. This role is independent of the Git provider roles of organization members.
+
+To review the additional permissions granted by the organization manager role, see the tables for each Git provider ([GitHub](#permissions-for-github), [GitLab](#permissions-for-gitlab), [Bitbucket](#permissions-for-bitbucket)).
+
+!!! important
+    Organization managers can access the **Policies** and **Integrations** settings sections of your organization and can therefore impact some repository settings for all repositories of your organization, even repositories that they can't access on the Git provider. However, they can't access the repositories themselves and can only see the repository names.
+
+### Assigning the organization manager role
+
+To assign the organization manager role:
+
+1.  Open your organization **Settings**, page **Roles and permissions**.
+
+1.  In the **Organization managers** area, use the search field to find the relevant organization member and click the member's name.
+
+    !!! note
+        You can only assign the organization manager role to [members of your organization](./managing-people.md#joining).
+
+    ![Security and risk management access management](images/roles-permissions-organization-manager-assign.png)
+
+### Revoking the organization manager role
+
+To revoke the organization manager role:
+
+1.  Open your organization **Settings**, page **Roles and permissions**.
+
+1.  In the **Organization managers** area, scroll the list to find the relevant user.
+
+1.  Click the **Revoke role icon** to the right of the user's name and confirm.
+
+## Configuring who can change the analysis configuration {: id="change-analysis-configuration"}
```

Comment:
Consider moving this section before the **The organization manager role**, for consistency with the UI.

Also, this title is very different from the UI label, **Repository management permissions**. Should we rename this section to **Configuring repository management permissions**, or similar?
Thoughts? 🤔